lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -346,7 +346,7 @@ e2function void entity:propSetFriction(number friction)
 end
 
 e2function number entity:propGetFriction()
-	if not PropCore.ValidAction(self, this, "friction") then return end
+	if not PropCore.ValidAction(self, this, "friction") then return 0 end
```

Comment:
Not sure if we should return 0 here as `GetFriction` can return any number for the most part.

Reply:
We could return `nan` in these cases, but returning 0 seems in line with what we do in plenty of other places in E2.